sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -852,7 +852,19 @@ void gnrc_ipv6_netif_init_by_dev(void)
             _add_addr_to_entry(ipv6_if, &addr, 64, 0);
 
         }
-
+#ifdef RIOT_STATIC_IPV6_LLADDR
```

Comment:
On second thought: This is very GNRC specific... why not rename it `GNRC_IPV6_STATIC_LLADDR` and add it (guarded with `#ifdef DOXYGEN`) to `net/gnrc/ipv6.h`?